scripts/shellcheck.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+EXCLUDE_RULES="SC1090,SC1091,SC2001,SC2064,SC2181,SC1117,SC2164"
+
+# Omitting:
+# - the `.git/` directory since its hooks won't pass # validation, and
+#   we don't maintain those scripts.
+# - the `.venv/` dir because we don't control files in there.
+# - ossec packages because there are a LOT of violations, and we have
+#   a separate issue dedicated to cleaning those up.
```

Comment:
(small nit, not worth fixing just this alone - comment slightly confusing since we don't have OSSEC packages in here)

Reply:
oops, meant to remove this.